gateways/cybersource/request_builders.go

```diff
@@ -169,10 +175,18 @@ func buildCaptureRequest(captureRequest *sleet.CaptureRequest) (*Request, error)
 	}
 	if captureRequest.ClientTransactionReference != nil {
 		request.MerchantDefinedInformation = append(request.MerchantDefinedInformation, MerchantDefinedInformation{
-			Key: "1",
+			Key:   "1",
 			Value: *captureRequest.ClientTransactionReference,
 		})
 	}
+	if captureRequest.Options[captureSequenceNumber] != nil && captureRequest.Options[totalCaptureCount] != nil {
```

Comment:
should we use the val, ok = map[key] pattern instead? A non-pointer interface might not give a nil value